crates/sui-graphql-rpc/src/types/transaction_block.rs

```diff
@@ -62,28 +53,87 @@ pub(crate) struct TransactionBlockFilter {
     pub transaction_ids: Option<Vec<String>>,
 }
 
-#[ComplexObject]
+#[Object]
 impl TransactionBlock {
     /// A 32-byte hash that uniquely identifies the transaction block contents, encoded in Base58.
-    /// This serves as a unique id for the block on chain
+    /// This serves as a unique id for the block on chain.
     async fn digest(&self) -> String {
-        self.digest.to_string()
+        Base58::encode(&self.stored.transaction_digest)
```

Comment:
Why is this encoded in Base58?

Reply:
All digests are Base58 encoded.  Previously `self.digest.to_string()` was doing the same thing (behind the scenes, it calls `Base58::encode`), but `self.stored.transaction_digest` is a raw byte array rather than an instance of the `Digest` type (which should be going away eventually), so we need to call it ourselves.